src/transformers/commands/add_new_model_like.py

```diff
@@ -1189,6 +1189,13 @@ def create_new_model_like(
             if "tokenization" not in str(f) and "processor" not in str(f) and "feature_extraction" not in str(f)
         ]
 
+    def disable_fx_test(filename: Path):
```

Comment:
What do you think of this @sgugger ?

The reason I added that is because `symbolic_trace` checks the model class before trying to trace the model to make sure it is supported. 
Because the tests are copied, if a new model is created from a supported model for symbolic tracing, the test file will contain something like `fx_ready = True` which will trigger the torch.fx tests, all of them failing because the model class is not in the list of the supported models. 
I do not think this is a good approach to automatically add the new model class to the supported models because the model implementation can be changed, so I thought that disabling the test and printing some message was a better option.

Reply:
Works for me!